src/components/AirdropWalletBalanceCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+
+import { CONST, AirdropWalletDetails, AirdropGoal } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+import { AirdropWalletBalance } from './AirdropWalletBalance';
+
+const i18n = require('../../loc');
+
+interface Props {
+  walletDetails: AirdropWalletDetails;
+}
+
+export const AirdropWalletBalanceCard: FC<Props> = ({ walletDetails }) => {
```

Comment:
I've looked again at designs it can be not so easy to mix carousel and details because carousel has what you will become, and details what you are. Please think it through, how those components can share common logic, threshold etc.

Reply:
Hmhm, this card is used in Carousel and then after airdrop is done in details, right? So diffing factor is `isAfterAirdrop`? Correct? 
So I think it `isAfterAirdrop ? reachedGoal : nextGoal`. Right?

Reply:
could work:)

Reply:
done